### DISCUSSION TITLE
Could you please let me know when Docker Hub will be updated to a newer version?

### DISCUSSION THREAD

**Discussion by 3419912225** [RocketChat/Rocket.Chat#33559] [Q&A]
Could you please update the Docker Hub version? Currently, it's still 6.11.1

**Comment by 3419912225** [comment]
I apologize for the inconvenience, as I am not able to package the GitHub version, I will have to wait for you, the author, to update the Docker Hub version.

**Comment by reetp** [comment]
I believe this is a internal issue. The image is voluntarily maintained but I am not sure thre is anyone working on it currently.

I am not sure whether it wil be resolved or not but will ask.

**Comment by rodrigok** [comment]
7.0 is available at Docker Hub https://hub.docker.com/layers/rocketchat/rocket.chat/7.0.0/images/sha256-39a7c8db640f4cbdfcf2575e01cf9584e3beae7d595a89d7b917f1127b3d4c86?context=explore 

The https://hub.docker.com/r/rocketchat/rocket.chat/ is managed by the company, the https://hub.docker.com/_/rocket.chat is maintained by Docker team
